kotlin/render-multilayer-symbols/src/main/java/com/esri/arcgisruntime/sample/rendermultilayersymbols/MainActivity.kt

```diff
@@ -0,0 +1,595 @@
+/* Copyright 2022 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.rendermultilayersymbols
+
+import android.graphics.Color
+import android.os.Bundle
+import android.util.Log
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment
+import com.esri.arcgisruntime.geometry.*
```

Comment:
let's try to keep these to single imports if possible. Helps direct users to which part of the API is being used exactly and helps the search function

Reply:
This is the result of using Android reformatting haha. Fixed this.